src/components/UI/navbar/index.js

```diff
@@ -55,6 +55,9 @@ const Navbar = () => {
             setUserData({
               userName: responseJson.username,
               firstName: responseJson.first_name,
+              profilePicture: responseJson.picture
+                ? responseJson.picture.url
+                : DEFAULT_AVATAR,
             });
           }
           return window.location.replace('https://my.realdevsquad.com/signup');
```

Comment:
I didn't get the workflow. Does it mean if the profilePicture is Default_AVATAR, it will get open in a new tab. But why open it in a new window in the first place until the user does not click on it.

Reply:
No I am asking to modify that return line 63. which replaces the current URL/ location. We can open in new tab because what if user is not logged in and this will redirect to `/signup` flow and ask the user to login. Then, we lost the crypto page because the site is opening in the same tab. 

What's your take on this ??

Reply:
Yeah got your point. well its the member site not crypto😅

Reply:
Ahh 😂 sorry, I was reviewing the Shubham's PR for improvement of navbar in crypto site, that's why mistook it.